app/controllers/starburst/announcements_controller.rb

```diff
@@ -1,3 +1,5 @@
+require_dependency 'starburst/application_controller'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.